qutebrowser/mainwindow/prompt.py

```diff
@@ -445,6 +447,21 @@ def prompt_item_focus(self, which):
         except UnsupportedOperationError:
             pass
 
+    @cmdutils.register(instance='prompt-container', scope='window',
+                       modes=[usertypes.KeyMode.prompt])
+    @cmdutils.argument('which', choices=['next', 'prev'])
+    def prompt_history(self, which):
+        """Shift the focus of the prompt file completion menu to another item.
+
+        Args:
+            which: 'next', 'prev'
+        """
+        assert self._prompt is not None
+        if which == 'prev':
+            self._prompt.history_prev()
```

Comment:
The prompt here isn't guaranteed to be a `FilenamePrompt`. You will need to raise `UnsupportedOperationError` for prompts which don't support this, similar to how e.g. `prompt_item_focus` / `item_focus` do.